lib/db/mysql.js

```diff
@@ -110,7 +110,7 @@ module.exports = function (log, error) {
     // Where  : name = $1
     var DB_METADATA = 'CALL dbMetadata_1(?)'
 
-    return mysql.readFirstResult(DB_METADATA, options.patchKey)
+    return mysql.readFirstResult(DB_METADATA, [options.patchKey])
```

Comment:
Happy if you'd like to change all of these in a separate patch, but probably best not to bundle these changes with this patch (ditto with all the `[id]` et al below too). Hope that's ok. To be honest, I  think `param` or `[param]` works fine with the driver so am not sure if this change gives us anything anyway.